package.json

```diff
@@ -2,31 +2,29 @@
   "name": "bobaserver",
   "version": "1.0.0",
   "description": "Server for BobaBoard",
-  "homepage": "https://docs.bobaboard.com/",
+  "homepage": "https://github.com/essential-randomness/bobaserver#readme",
   "bugs": {
-    "url": "https://github.com/BobaBoard/boba-backend/issues"
+    "url": "https://github.com/essential-randomness/bobaserver/issues"
   },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/BobaBoard/boba-backend.git"
+    "url": "git+https://github.com/essential-randomness/bobaserver.git"
   },
   "license": "MIT",
-  "author": "BobaBoard Team",
+  "author": "",
   "main": "dist/server/index.js",
   "scripts": {
-    "build": "tsc && tsc-alias && copyfiles .env .env.prod firebase-sdk.json server/**/*.sql ./dist",
+    "db:start": "docker compose down && docker compose up",
     "deploy": "gcloud config set project bobaboard && gcloud app deploy --version live",
-    "dev:watch": "cross-env DEBUG=bobaserver:*,-*info DEBUG_DEPTH=10 nodemon --watch . --exec node -r tsconfig-paths/register -r ts-node/register server/index.ts -e yaml,ts,sql",
     "open-api:docker": "docker compose -f docker-compose.open-api.test.yml run --rm test",
     "open-api:generate": "node -r dotenv/config -r ts-node/register -r tsconfig-paths/register utils/generate-spec-file.ts dotenv_config_path=./.env.prod",
     "open-api:typescript": "yarn run open-api:generate && openapi-zod-client ./open-api-spec.json -o ./types/open-api/generated/schemas.ts --export-schemas --template ./types/open-api/generated/schemas.hbs --with-alias &&  openapi-zod-client ./open-api-spec.json -o ./types/open-api/generated/types.ts --export-schemas --template ./types/open-api/generated/types.hbs --with-alias",
     "open-api:validate": "yarn run open-api:generate && validate-api open-api-spec.json && openapi-examples-validator open-api-spec.json --ignore-formats color quill-delta && openapi lint open-api-spec.json --skip-rule=no-ambiguous-paths",
-    "stage": "gcloud config set project bobaboard && gcloud app deploy --quiet --version staging --no-promote",
-    "start": "cross-env DEBUG=bobaserver:*,-*info node -r dotenv/config dist/server/index.js dotenv_config_path=./.env.prod",
-    "start-db": "docker compose down && docker compose up",
```

Comment:
Should we consider changing start-db to be like `node -e \"console.error('This command has been deprecated. Use db:start now.')\"`, at least for now? Same for `dev:watch`.